tests/Unit/App/Http/Middleware/RequestLoggerTest.php

```diff
@@ -2,30 +2,23 @@
 
 use App\Http\Middleware\RequestLogger;
 use Illuminate\Http\Request;
+use Psr\Log\LoggerInterface;
 use Tests\TestCase;
 
 class RequestLoggerTest extends TestCase
 {
     public function test_log_requests_to_file()
     {
-        $logger_middleware = new RequestLogger();
-
+        $logger = $this->prophesize(LoggerInterface::class);
```

Comment:
We no longer need to test that the logger actually works, we can trust that monolog actually works (it has many, many tests afterall). As such, we just created a fake an inject that in, all we need to do is ensure that the logger is called with the correct params.